datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -18,6 +18,13 @@
 TESTING_PLUGIN = 'DDEV_TESTING_PLUGIN'
 SKIP_ENVIRONMENT = 'DDEV_SKIP_ENV'
 
+JMX_TO_INAPP_TYPES = {
+    'counter': 'gauge',  # JMX counter -> DSD gauge -> in-app gauge
+    'monotonic_count': 'rate',  # JMX monotonic_count -> DSD count -> in-app rate
```

Comment:
+ `'rate': 'gauge'` ?

Reply:
There is actually no `rate`, even `counter` is based on a comment: https://github.com/DataDog/jmxfetch/blob/f39e75a8a3d5c83890d01f6752cea4dc0aef679c/src/main/java/org/datadog/jmxfetch/reporter/Reporter.java#L122

It seems to be a bit unfortunate to accept anything as default. Maybe something like this is better:

```
            if ("gauge".equals(metricType) || "histogram".equals(metricType)) {
                ...
            } else if ("monotonic_count".equals(metricType)) {
                ...
            } else if ("counter".equals(metricType)) {
                ...
            } else {
                // raise Exception or warning
            }
```

Reply:
`rate` seems to be documented here: https://docs.datadoghq.com/integrations/java/?tab=host#the-attribute-filter

but I think it's better if we stick to one type instead of having multiple types doing the same thing.

I don't have strong opinion about adding or not `'rate': 'gauge'`